src/components/PdfButton/PdfButton.cy.tsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import PdfButton from "@/components/PdfButton/PdfButton";
+import ShippingLabelsPdf, { ParcelClients } from "@/pdfs/ShippingLabels/ShippingLabelsPdf";
+
+const downloadsFolder = Cypress.config("downloadsFolder");
+const fileName = "ShippingLabels.pdf";
+const parcelClientsData: ParcelClients[] = [
+    {
+        packing_datetime: "19/06/2023",
+        collection_centre: "Delivery",
+        collection_datetime: "31/07/2023, 14:00",
+        voucher_number: "VOUCHER-NUMBER-2",
+        full_name: "Harry Potter",
+        phone_number: "07987654321",
+        address_1: "Hogwarts Castle",
+        address_2: "",
+        address_town: "Scottish Highlands",
+        address_county: "United Kingdom",
+        address_postcode: "YE11OW",
+        delivery_instructions: "Owls only no floo powder please.",
+        index: 1,
```

Comment:
How come we need index and total here? For any map function you should be able to do `(content, index) => {...}` or similar. (May figure this out later but haven't looked through the code.

Reply:
Have looked through, can't see why we need index and total but also may just be me missing something

Reply:
Before this I was using the 5 per page thing which means that I needed the data to be a 2D array hence that structure but now you are correct I dont need that index now. Nice catch!